src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -143,7 +143,7 @@
     }
   }
 
-  .isRTL {
+  &.isRTL {
```

Comment:
this fixes the menu and settings buttons too close, and the menu opening on the wrong direction